### DISCUSSION TITLE
[SUGGESTION] Select parent

### DISCUSSION THREAD

**Discussion by ddiaz2380** [TiagoSilvaPereira/vemto-issues#143] [Ideas & Suggestions]
Could you add parent selection, for relations? It would be great this function.

**Comment by TiagoSilvaPereira** [comment]
Hi @ddiaz2380 could you please share more details about the suggestion?

**Comment by ddiaz2380** [comment]
I mean nested select, example: country, city, state. That the selections are inherited, if I put Brazil, bring me only the municipalities of Brazil and when they are in states, the same thing happens.
Thanks for listening to suggestions, your tool is great.

**Comment by TiagoSilvaPereira** [comment]
Got it... That is a great suggestion bro, thanks
